lms/djangoapps/instructor/enrollment.py

```diff
@@ -219,9 +220,11 @@ def reset_student_attempts(course_id, student, module_state_key, delete_module=F
         submissions.SubmissionError: unexpected error occurred while resetting the score in the submissions API.
 
     """
+    user_id = anonymous_id_for_user(student, course_id)
+    submission_cleared = False
     try:
+        block = get_module_for_student(context_user, module_state_key)
```

Comment:
This change is the key of this PR, all the blocks of code below are the ORA delete state code changes that got reverted when this bug was found.